.gitignore

```diff
@@ -11,7 +11,8 @@
 # Ignore the default SQLite database.
 /db/*.sqlite3
 /db/*.sqlite3-journal
-
+Gemfile
```

Comment:
You can't ignore the Gemfile and Gemfile.lock, this is required to setup the project for all users

Reply:
I have configure .gitignore but i don't now why he doesn't ignore it.

Reply:
I have not edit the `config/locales/app.public.en.yml` yes but the code is juste `about_the_fablab: "About the {NAME}"  # messageFormat interpolation` have not parameter for male or female..

Reply:
Ah et je viens de voir sur une issue que tu étais Français, on peut parler français alors ça sera plus simple haha.